core/src/main/java/org/elasticsearch/monitor/Probes.java

```diff
@@ -0,0 +1,20 @@
+package org.elasticsearch.monitor;
+
+import java.lang.management.OperatingSystemMXBean;
+import java.lang.reflect.Method;
+
+public class Probes {
+    public static short getLoadAndScaleToPercent(Method method, OperatingSystemMXBean osMxBean) {
```

Comment:
It's used in two places, `OsProbe` and `ProcessProbe`. I moved some logic that was going to be duplicated into the two places to this method.